.github/workflows/5_builderprecompiled_server-images.yml

```diff
@@ -20,16 +20,25 @@ on:
         options:
           - deb
           - rpm
+      architecture:
+        description: |
+          Architecture of the package [amd64, x86_64, arm64]
+        type: choice
+        options:
+          - amd64
+          - x86_64
+          - arm64
```

Comment:
ditto

Reply:
Fixed in c8a4388ee3b0326508f38a0ab706ca3fdff3825c